.travis.yml

```diff
@@ -2,7 +2,7 @@ language: java
 
 jdk:
   - openjdk7
```

Comment:
Why not openjdk8, too?

P.S. Since most of these commits were not mine, I'll be adding comments, too.